test/tasks/example/sep/008-opcm-upgrade-stage1/config.toml

```
@@ -0,0 +1,14 @@
+templateName = "OPCMUpgradeV400"
+
+[[l2chains]]
+chainId = 11155420
+name = "OP Sepolia Testnet"
+
+[[opcmUpgrades]]
+chainId = 11155420
+absolutePrestate="0x0000000000000000000000000000000000000000"
+expectedValidationErrors = ""
+
+[addresses]
+OPCM = "0x355fd2a98fc11f5e4a91d9aa4c0041d252a77583" # Coming from interop warrom
+StandardValidatorV300 = "0x2d56022cb84ce6b961c3b4288ca36386bcd9024c" # Sepolia https://github.com/ethereum-optimism/optimism/blob/f79ed8b9c9cbdbf8bb492074f3f98da7f072e21a/op-validator/pkg/validations/addresses.go#L30
```

We should remove the link to the go file and rename the variable. Also is the address correct? The one we deployed today is `0xd0233AA63b11c2b93F68E306A16f2786Fb11D0d1`.

```suggestion
StandardValidatorV400 = "0x2d56022cb84ce6b961c3b4288ca36386bcd9024c" # Sepolia
```